src/test/java/org/opensearch/security/privileges/ActionPrivilegesTest.java

```diff
@@ -0,0 +1,588 @@
+/*
```

Comment:
Lets move these tests under the newer [integrationTest](https://github.com/opensearch-project/security/tree/main/src/integrationTest) framework